addOns/spider/CHANGELOG.md

```diff
@@ -6,13 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-### Added
-- Functionality to parse URLs from href attributes in SVG files (Issue 4984).
-- Support for automation monitor tests.
+### Functional Improvements Compared to Previous Core Release
 
```

Comment:
Alternatively, we could simply link to the blog post here but this works for me too.

Reply:
Better have something in the add-on itself.